### ISSUE TITLE
How to install broken scoop installation

### ISSUE THREAD

**Issue by afoglia** [ScoopInstaller/Scoop#3846]
I installed scoop a couple of days ago to install ag. Today, when I tried to install vim, I got a bunch of errors. Now when I run scoop <anything> I get the error:

> scoop help
& : The term
'C:\Users\afoglia\scoop\shims\..\apps\scoop\current\bin\scoop.ps1' is not
recognized as the name of a cmdlet, function, script file, or operable
program. Check the spelling of the name, or if a path was included, verify
that the path is correct and try again.
At C:\Users\afoglia\scoop\shims\scoop.ps1:3 char:71
+ ... on.expectingInput) { $input | & $path  @args } else { & $path  @args  ... +                                                             ~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\Users\afogli...t\bin\scoop.p
   s1:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

So how do I get rid of this broken manager? Just delete the scoop directory and remove it from my PATH?

**Comment by carstenuhlig** [comment]
I think in the wiki there is a section explaining in case it is broken: https://github.com/lukesampson/scoop/wiki/Uninstalling-Scoop

That worked for me. But I came across a different problem..., which I could not solve via deleting simply the scoop folder. Let me know if that helped.

**Comment by 2bon** [comment]
choco upgrade docker-desktop

**Comment by rashil2000** [comment]
> choco upgrade docker-desktop

?

**Comment by danielb2** [comment]
```
scoop uninstall scoop
& : The term 'C:\Users\nefar\scoop\shims\..\apps\scoop\current\bin\scoop.ps1' is not recognized as the name of a
cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify
that the path is correct and try again.
At C:\Users\nefar\scoop\shims\scoop.ps1:3 char:72
+ ... on.ExpectingInput) { $input | & $path  @args } else { & $path  @args  ...
+                                                             ~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\Users\nefar\...t\bin\scoop.ps1:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
```

Would love to know how to do this. I installed chocolatey and there were conflicting execution policies, and now I don't know how to fix. is it not possible to have them side-by-side ?

**Comment by rashil2000** [comment]
It's possible to have them side by side.

You can just delete the scoop.ps1 shim and reinstall Scoop itself

**Comment by danielb2** [comment]
How to install them properly then? Choco first then scoop?

**Comment by rashil2000** [comment]
You can install in any order. They're not supposed to conflict.

**Comment by danielb2** [comment]
> You can install in any order. They're not supposed to conflict.

they do tho. I got the above error right away. As I could understand it, it's the execution policies that had a conflict. Course I could be wrong. But after installing chocolatey, scoop didn't work anymore complaining about policy.

**Comment by rashil2000** [comment]
Where's the error about execution policy?

**Comment by danielb2** [comment]
The error above left me with an unusable scoop so I purged chocolatey  and reinstalled scoop and got it to work after I posted the original question here 4 days ago. IOW, I've fixed the problem so I'm not in a place where I can check anymore.

I literally just installed scoop first and then chocolatey on a fresh machine (account will probably do it too). Windows isn't my main platform as a developer, so I use it less.  I don't have more info than that.
